app/services/translation_service/api/translations.rb

```diff
@@ -0,0 +1,58 @@
+module TranslationService::API
+
+  class Translations
+
+    TranslationStore = TranslationService::Store::Translation
+
+    ## GET /translations/:community_id/
+    def get(community_id, find_params = {})
```

Comment:
What about using EntityUtils.define_builder to document and validate the find_params input hash?